waku/v2/node/wakuoptions.go

```diff
@@ -80,10 +80,8 @@ type WakuNodeParameters struct {
 	enableStore     bool
 	messageProvider store.MessageProvider
 
-	rendezvousNodes        []multiaddr.Multiaddr
-	enableRendezvousServer bool
-
-	rendezvousDB *rendezvous.DB
+	enableRendezvous bool
```

Comment:
Interesting thought. If we do that for rendezvous, we should probably do the same for the other discovery protocols: discv5 and peer exchange. If we decide to follow this route, it would make sense to ensure that nwaku's defaults for these protocols also match go-waku